apps/webapp/components/layout/footer/newsletter.tsx

```diff
@@ -91,7 +91,7 @@ export default function Newsletter({ lang }: LangProp) {
   }, [email])
 
   return (
-    <div className="relative overflow-hidden" id="newsletter">
+    <div className="relative m-10 overflow-hidden" id="newsletter">
```

Comment:
Please change the className `m-10` to `my-10`. It seems that is making the section shrink on mobile.